{"name": "test_login_with_validemail_and_invalidpassword_crenditial", "status": "broken", "statusDetails": {"message": "AttributeError: 'LoginPage' object has no attribute 'retrived_warning_message'. Did you mean: 'retrieve_warning_message'?", "trace": "self = <test.test_login.TestLogin object at 0x000001CBE79285F0>\n\n    def test_login_with_validemail_and_invalidpassword_crenditial(self):\n        home_page = HomePage(self.driver)\n        home_page.click_on_my_account_drop_menu()\n        login_page =home_page.select_login_option()\n        login_page.enter_email_address(\"swapnild02@gmail.com\")\n        login_page.enter_password(\"12ygt\")\n        login_page.click_on_login_button()\n        expected_text = \"Warning: No match for E-Mail Address and/or Password.\"\n>       assert expected_text.__eq__(login_page.retrived_warning_message())\nE       AttributeError: 'LoginPage' object has no attribute 'retrived_warning_message'. Did you mean: 'retrieve_warning_message'?\n\ntest\\test_login.py:47: AttributeError"}, "attachments": [{"name": "stdout", "source": "a1ffa6b1-21cf-429d-9461-2c07b4180f72-attachment.txt", "type": "text/plain"}], "start": 1733160317236, "stop": 1733160318958, "uuid": "382773ca-7f9a-4fc6-88d3-0efb5b99d772", "historyId": "3aef4f1c921b724d168ef8281cb0dc88", "testCaseId": "3aef4f1c921b724d168ef8281cb0dc88", "fullName": "test.test_login.TestLogin#test_login_with_validemail_and_invalidpassword_crenditial", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Agni"}, {"name": "thread", "value": "7036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_login"}]}